pkg/api/v1beta1/defaults.go

```diff
@@ -70,6 +70,15 @@ func init() {
 				obj.DNSPolicy = DNSClusterFirst
 			}
 		},
+		func(obj *Pod) {
+			for _, v := range obj.DesiredState.Manifest.Volumes {
+				if v.Source.PersistentVolumeClaimVolumeSource != nil &&
+					v.Source.PersistentVolumeClaimVolumeSource.PersistentVolumeClaimRef != nil {
+					// ClaimRefs will be posted with a name to the user's claim, but we don't want/need them to set a namespace
```

Comment:
Agreed.  the PersistentVolumeClaimVolumeSource (say that 5x fast) still requires the pod's namespace to lookup the PV by name, but that is addressed elsewhere in this thread and the potential inclusion of pod or namespace as an argument to volume.NewBuilder.